.github/workflows/public-api-checks.yml

```diff
@@ -0,0 +1,24 @@
+name: Public API check
+on: pull_request
+jobs:
+
+  public-api-check:
+    name: Public API Check
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Get changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v35
+      with:
+        files: library/public/*.h
+
+    - name: Warn on PR
+      uses: thollander/actions-comment-pull-request@v2
+      if: steps.changed-files.outputs.all_changed_and_modified_files != ''
+      with:
+        message: You are modifying libf3d public API! :warning:Please update bindings accordingly:warning:!
```

Comment:
Maybe give an exhaustive list of bindings (python, java, js) and inform the user where the binding files are located.